src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -60,7 +60,7 @@ object TrialService {
 
 final class TrialService
   (val samDao: SamDAO, val thurloeDao: ThurloeDAO, val rawlsDAO: RawlsDAO,
-   val trialDAO: TrialDAO, val googleDAO: GoogleServicesDAO, projectManager: ActorRef)
+   val trialDao: TrialDAO, val googleDAO: GoogleServicesDAO, projectManager: ActorRef)
```

Comment:
needed to rename `trialDAO` to `trialDao` due to naming conflicts between traits - unfortunately this affects many lines of code in the diff.